frame/nomination-pools/src/lib.rs

```diff
@@ -411,6 +411,21 @@ enum AccountType {
 	Reward,
 }
 
+// Account to increse the bond of a member
+#[derive(Encode, Decode, MaxEncodedLen, Clone, Copy, Debug, PartialEq, Eq, TypeInfo)]
+pub enum BondExtraSource {
+	// Action to be taken by the bonded member of a pool
+	Origin,
+	// Action to be taken by the pool operator on the member's behalf
+	Operator,
+}
```

Comment:
```suggestion
pub enum RewardClaim {
	/// Only `self` i.e. the pool member themself can claim their reward. 
	Self,
	/// Anyone can claim the reward of this member. 
	Permissionless,
}
```

Reply:
`Self` is a keyword, I suggest keeping it as `Origin`

Reply:
`Permissioned`? 

`Origin` has a different meaning in substrate. It doesn't mean myself at all. An origin is the originator of any message. It can be myself, it can be someone else, it can be a governance body, etc.

Reply:
`Permissioned` ✅